spikeinterface/postprocessing/isi.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+
+try:
+    import numba
+    HAVE_NUMBA = True
+except ModuleNotFoundError as err:
+    HAVE_NUMBA = False
+
+
+def compute_ISI_from_spiketrain(spike_train: np.ndarray, max_time: int,
+                                bin_size: int, sampling_f: float):
+    """
+    Computes the Inter-Spike Intervals histogram from a given spike train.
+
+    This implementation only works if you have numba installed, to accelerate the
+    computation time.
+
+    Parameters
+    ----------
+    spike_train: np.ndarray
+        The ordered spike train to compute the ISI.
+    max_time: int
+        Compute the ISI from 0 to +max_time (in sampling time).
+    bin_size: int
+        Size of a bin (in sampling time).
+    sampling_f: float
+        Sampling rate/frequency (in Hz).
+
+    Returns
+    -------
+    tuple (ISI, bins)
+    ISI: np.ndarray[int64]
+        The computed ISI histogram.
+    bins: np.ndarray[float64]
+        The bins for the ISI histogram.
+    """
+    if not HAVE_NUMBA:
+        print("Error: numba is not installed.")
+        print("compute_ISI_from_spiketrain cannot run without numba.")
+        return 0
+
+    return _compute_ISI_from_spiketrain(spike_train.astype(np.int64), max_time, bin_size, sampling_f)
+
+if HAVE_NUMBA:
+    @numba.jit((numba.int64[::1], numba.int32, numba.int32, numba.float32),
+               nopython=True, nogil=True, cache=True)
+    def _compute_ISI_from_spiketrain(spike_train, max_time, bin_size, sampling_f):
+        n_bins = int(max_time / bin_size)
+
+        bins = np.arange(0, max_time+bin_size, bin_size) * 1e3 / sampling_f
+        ISI = np.zeros(n_bins, dtype=np.int64)
+
+        for i in range(1, len(spike_train)):
+            diff = spike_train[i] - spike_train[i-1]
+
+            if diff >= max_time:
+                continue
+
+            bin = int(diff / bin_size)
+            ISI[bin] += 1
+
+        return ISI, bins
+
+
+
+def compute_ISI(sorting, window_ms: float = 50.0, bin_ms: float = 1.0,
```

Comment:
```suggestion
def compute_isi_histograms(sorting, window_ms: float = 50.0, bin_ms: float = 1.0,
```

Reply:
@Dradeliomecus this is actually computing the histogram, let's make it clear with the naming ;)